src/epics/router.ts

```diff
@@ -188,6 +229,53 @@ const handleRouter: Epic = (action$, store) => action$.ofType(types.handleRouter
     })
 })
 
+function parseFiltersFromString(defaultFilters: string) {
+    const result: BcFilter[] = []
+    const urlParams = qs.parse(defaultFilters)
+    Object.keys(urlParams).forEach((param) => {
+        const [fieldName, type] = param.split('.')
+        if (fieldName && type && urlParams[param]) {
+            let value = urlParams[param]
+            if (isManyValueFilterType(type)) {
+                try {
+                    value = JSON.parse(value)
+                } catch (e) {
+                    console.warn(e)
+                }
+                value = Array.isArray(value) ? value : []
+            }
+            result.push({
+                fieldName,
+                type: type as FilterType,
+                value
+            })
+        }
+    })
+    return result.length ? result : null
+}
+
+function isManyValueFilterType(type: string) {
```

Comment:
It's possible to inline that check as it's pretty basic and not reused anywhere

Reply:
fixed